images/scripts/virt-install-rhel4edge

```diff
@@ -35,6 +35,11 @@ url=$3
 
 base=$(dirname $0)
 
+fleet_ks_file="$(basename $url).ks"
+# Extract and fix the kickstart for rhel4edge. Rootpw has to be defined
```

Comment:
This should be marked as "HACK:" and we should report this as a bug. This is obviously broken for any customer who wants to do a non-interactive installation, i.e. all of them (this is IoT, after all). If you want to do this, please do, otherwise I'm happy to do it as well if you got fed up with it :grin:

Reply:
@martinpitt many thanks for suggestions.
Doc fixed. If you know where to file the bug, please do so. I'm not sure which component does that, after discussions with many folks, seems no anaconda, maybe image-builder, but maybe something totally else, like some kickstart generator what is used in `console.`

Reply:
I don't know either, but let's perhaps start with https://issues.redhat.com/projects/THEEDGE/issues and let folks there reassign if necessary?

Reply:
Okay, I'll file  the issue. Thanks a lot.

Reply:
created: https://issues.redhat.com/browse/THEEDGE-3825

Reply:
Cheers! I also followed up there. I'll unstick the hanging image build, and will then push the rebuild with some editorial changes to the commit message.